### PR TITLE
Maps Centroid

### PR DESCRIPTION
**Pull request by anissa-agahchen**
# Overview

## Links to jira tickets

-https://quartech.atlassian.net/browse/BHBC-1567

### CODE REVIEW

File: api/src/paths/public/projects.ts [hunk 1]
```diff
@@ -54,6 +57,8 @@ export function getPublicProjectsList(): RequestHandler {
     const connection = getAPIUserDBConnection();
 
     try {
+      await connection.open();
```
Comment (NickPhura) [inline api/src/paths/public/projects.ts]: There are 2 connection.open calls

Comment (NickPhura) [reply api/src/paths/public/projects.ts]: Here and line 68

File: api/src/paths/public/projects.ts [hunk 2]
```diff
@@ -62,24 +67,25 @@ export function getPublicProjectsList(): RequestHandler {
...
-      );
+      const response = await connection.query(getProjectListSQLStatement.text, getProjectListSQLStatement.values);
 
       await connection.commit();
 
-      let rows: any[] = [];
-
-      if (getProjectListResponse && getProjectListResponse.rows) {
-        rows = getProjectListResponse.rows;
+      if (!response || !response.rows) {
```
Comment (NickPhura) [inline api/src/paths/public/projects.ts]: nit: can remove the `!response` check.  

I had the incorrect type definition before (in my db wrapper) which indicated that the response could be null, but in reality the underlying library will never return null.

File: api/src/paths/public/projects.ts [hunk 3]
```diff
@@ -62,24 +67,25 @@ export function getPublicProjectsList(): RequestHandler {
...
+        return res.status(200).json(null);
       }
 
-      const result: any[] = _extractProjects(rows);
+      const project_rows = response.rows;
+
+      console.log(' project list from DB', project_rows);
```
Comment (NickPhura) [inline api/src/paths/public/projects.ts]: a few console logs to remove

File: api/src/queries/public/project-queries.ts [hunk 4]
```diff
@@ -348,7 +348,7 @@ export const getPublicProjectListSQL = (): SQLStatement | null => {
     left outer join permit as pp
       on p.project_id = pp.project_id
     where
-      p.publish_timestamp is not null
+      1=1
```
Comment (NickPhura) [inline api/src/queries/public/project-queries.ts]: Can just remove the where entirely

File: api/src/paths/public/projects.ts (hunk 1 above)
Comment (anissa-agahchen) [reply api/src/paths/public/projects.ts]: nice catch ... thanks

**Review by NickPhura — APPROVED** [review]

**Pull request merged**